spring-cassandra/src/test/java/org/baeldung/springcassandra/config/CassandraContainerInitializer.java

```diff
@@ -0,0 +1,40 @@
+package org.baeldung.springcassandra.config;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Session;
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.testcontainers.containers.CassandraContainer;
+import org.testcontainers.junit.jupiter.Container;
+
+public abstract class CassandraContainerInitializer {
+
+    private static final String KEYSPACE_NAME = "test";
+
+    @Container
+    public static final CassandraContainer cassandra = (CassandraContainer) new CassandraContainer("cassandra:3.11.2")
```

Comment:
Should be `CASSANDRA` as it's static final. Also, shouldn't really be `public`

Reply:
I left this lowercase intentionally. Since the variable here is not a primitive or a String, it is a pointer, not a constant. Do you agree with this convention or should I make it uppercase?

Reply:
@ashleyfrieze can I get your input on this?